generators/app/templates/_package.json

```diff
@@ -26,7 +26,7 @@
     "mkdirp": "0.5.1",
     "semver": "5.3.0",
     "shelljs": "0.5.3",
-    "yeoman-generator": "1.1.1"
+    "yeoman-generator": "^2.0.0"
```

Comment:
Please fix all package versions because of our [Policy 4](http://www.jhipster.tech/policies/).

Reply:
@PierreBesson ,Thanks for Pointing it out.